pegasus-common/src/main/java/com/linkedin/common/stats/LongTracking.java

```diff
@@ -80,22 +62,21 @@ public LongTracking(final int maxCapacity, int initialCapacity, double growthFac
     {
       growthFactor = DEFAULT_GROWTH_FACTOR;
     }
+
     _buffer = new long[initialCapacity];
     _bufferSize = initialCapacity;
     _initialCapacity = initialCapacity;
     _growthFactor = growthFactor;
     _maxCapacity = maxCapacity;
+    _simpleLongTracking = new SimpleLongTracking();
```

Comment:
Wondering what's the difference between doing this and let this class extend the SimpleLongTracking.java, and use super.xxx() instead. Which one is a better option?

Reply:
I personally prefer delegation over inheritance, because I would like to separate out the basic tracking logic from percentile tracking logic. The SimpleLongTracking instance acts just like a blackbox. I think the difference is pretty subtle here though. Feel free to share your opinion on this.

Reply:
OK, I don't have a strong preference here